bin/selfupdate.py

```diff
@@ -21,7 +21,8 @@
 
 
 class UpdateError(Exception):
-    pass
+    def __init__(self, message=""):
+    	self.message = message
```

Comment:
Wouldn't it be better to change the code that relies on the `message` attribute instead? If you want to display an exception, it's best to use `str(exc)`, so the name of the exception is included. If you want to leave the exception name out, you can use `exc.args[0]` (which works in Python 2 and 3).

Reply:
@dgelessus thanks for the tip :)

Reply:
__HINT:__ Undo these modifications and then in __atom system/shruntime.py__ change all instances of __e.message__ to __e.args[0]__